py3status/modules/process_status.py

```diff
@@ -7,6 +7,7 @@
     format_running: what to display when process running
     format_not_running: what to display when process is not running
     process: the process name to check if it is running
+    full: use the full command line to match the process
```

Comment:
> use the full command line to match the process

`if True match against the full command line not just the process name`

feels clearer to me